src/Utils.php

```diff
@@ -300,13 +314,13 @@ public static function jsonEncode($value, int $options = 0, int $depth = 512): s
      * Wrapper for the hrtime() or microtime() functions
      * (depending on the PHP version, one of the two is used)
      *
-     * @return float|mixed UNIX timestamp
+     * @return float UNIX timestamp
      *
      * @internal
      */
-    public static function currentTime()
+    public static function currentTime(): float
     {
-        return \function_exists('hrtime') ? \hrtime(true) / 1e9 : \microtime(true);
+        return (float) \function_exists('hrtime') ? \hrtime(true) / 1e9 : \microtime(true);
```

Comment:
When is this not a float?

Reply:
[`hrtime` documentation:](https://www.php.net/manual/en/function.hrtime.php) 

> Returns an array of integers in the form [seconds, nanoseconds], if the parameter get_as_number is false. Otherwise the nanoseconds are returned as integer (64bit platforms) or float (32bit platforms).

So, returns non-float when hrtime exists and on a 64 bit platform

Reply:
Cool